docs/estela/installation/main-guide.md

```diff
@@ -0,0 +1,224 @@
+  web interface. The SMTP server can be dedicated, e.g.
+  [SES](https://aws.amazon.com/ses/); or public, e.g.
+  [Google SMTP server](https://support.google.com/a/answer/176600?hl=en).
+
+### Local Deployment of Resources
+
+All the named resources (except the _Document Oriented Database_ and the _SMTP Email Server_)
+can be started locally by running:
+
+```bash
+$ make resources
+```
+
+To allow the use of images stored in the local container registry, you need to
+add the following line to the Docker daemon 
+[config file](https://docs.docker.com/config/daemon/#configure-the-docker-daemon).
+And then restart Docker.
+
+```json
+{
+	...
+	"insecure-registries" : [ "<HOST_IP>:5000" ]
+	...
+}
+```
+Where _<HOST\_IP>_ is equal to the result of:
+
+```bash
+$ minikube ssh 'grep host.minikube.internal /etc/hosts | cut -f1'
+```
+
+The MongoDB database can be deployed on MongoDB Atlas for
+[free](https://www.mongodb.com/free-cloud-database).
+To configure the use of a public SMTP server, please read the following
+[guide](https://kinsta.com/blog/gmail-smtp-server/). Nothing needs to be configured,
+just use the Google services and your Gmail account credentials.
+
+## Environment Variables
+
+Refer to the Helm Chart
+[variables guide]({% link estela/installation/helm-variables.md %}),
+and complete the `values.yaml` file with the 
+appropriate values.
+
+## Helm Deployment
+
+The images of each of the estela modules must be built and uploaded to the Docker
+Container Registry, make sure to do this step before installing the Helm application.
+
+If you are using a local registry, you can build and upload the images by running:
+
+```bash
+$ make images
+```
+
+### Installing the Helm application
+
+If you are using local resources, a `.env` file should have been created, if not, 
+create a new one. Open the file and add the following lines:
+
+```
+RELEASE_NAME=<RELEASE_NAME>
+NAMESPACE=<NAMESPACE>
+```
+
+You can use any value for `<RELEASE_NAME>` and `<NAMESPACE>`. However, to avoid writing 
+the namespace every time you use kubectl, you can set `NAMESPACE=default`.
+
+1. Install the helm application:
+
+   ```
+   $ make install
+   ```
+
+2. If you are using Minikube, you need an external IP for the Django service, please 
+   run this command in a new terminal:
+
+   ```
+   $ minikube tunnel
+   ```
+
+3. Update the application with the value of the Django service external IP:
+
+   ```
+   $ make update-api-ip
+   ```
+
+5. Now, you can perform the migrations and create a super user for Django admin:
+
+   ```
+   $ make makemigrations
+   $ make migrate
+   $ make createsuperuser
+   ```
+
+6. Once you do the migrations, restart the `estela-celery-beat` deployment:
+
+   ```bash
+   $ make restart-celery-beat
+   ```
+
+   The application is ready!
+
+### Uninstalling the Helm application
+
+To uninstall estela, just run:
+
+```
+$ make uninstall
+```
+
```

Comment:
Two things are missing for deployments to work locally:
1. You need to create a public bucket in MinIO with the name specified in the `BUCKET_NAME_PROJECTS` variable
2. You need to create a superuser called `deploy_manager`.

Reply:
Please, can you review and/or modify the MinIO guide? Maybe I missing something important.